src/components/details-card-loading/index.tsx

```diff
@@ -0,0 +1,10 @@
+import React from "react";
+import "./style.css";
+
+export const DetailsCardLoading: React.FC = () => {
```

Comment:
nos projetos da taq, esses componentes de Shimmer:
```suggestion
export const DetailsCardShimmer: React.FC = () => {
```
não tá errado o que vc fez tá? kkk é só pra vc conhecer mesmo 👍

Reply:
Bacana! Não conhecia nem essa palavra hahahaha

E tranquilo, quando mais coisa tiver para passar melhor! 👍🏻